### PR TITLE
Development branch of app

### PR DESCRIPTION
**Pull request by mamankhan99**

### CODE REVIEW

**Comment by mamankhan99** [comment]
Major functional requirements are complete

File: src/components/profile/Profile.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                            </div>
+                        </div>
+                        <div>
+                            <p>
+                                {`Bio: ${targetUser.profile.about}`}
+                            </p>
+                            {
+                                isUserProfile &&
+                                <Button 
+                                    href='/edit_profile'
```
Comment (abbas-nazar) [inline src/components/profile/Profile.js:50]: use constant

File: src/components/question/AddQuestion.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    const history = useHistory();
+
+    const state = useSelector(state => state.question);
+    
+    const tags_state = useSelector(
+        state => state.tag
+    );
+    const question_state = useSelector(
+        state => state.question
+    );
```
Comment (abbas-nazar) [inline src/components/question/AddQuestion.js:22]: add a container for this component

File: src/components/routes/PrivateContent.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import Profile from '../profile/Profile';
+import EditProfile from '../profile/EditProfile';
+import AddQuestion from '../question/AddQuestion';
+
+const PrivateContent = (props) => {
+    return (
+        <Box>
+          <Navbar/>
+          <Switch>
+            <PrivateRoute exact path="/add_question" component={AddQuestion}/>
```
Comment (abbas-nazar) [inline src/components/routes/PrivateContent.js:16]: use constants

File: src/redux/ducks/authentication.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import { LOGIN_URL } from "../../utils/urls";
+
+const SIGN_IN_REQUEST = 'stackoverflow/authentication/sign_in_request';
+const SIGN_IN_SUCCESS = 'stackoverflow/authentication/sign_in_success';
+const SIGN_IN_FAILURE = 'stackoverflow/authentication/sign_in_failure';
+const SIGN_OUT = 'stackoverflow/authentication/sign_out';
```
Comment (abbas-nazar) [inline src/redux/ducks/authentication.js:7]: move all action types to one common constant file

File: src/redux/ducks/comment.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
+// import axios from 'axios';
+// import { DEFAULT_MODEL_TYPE } from '../../utils/constants';
+// import { COMMENT_API_URL } from "../../utils/urls";
+
+// const GET_COMMENT_REQUEST = 'stackoverflow/comment/get_comment_request';
```
Comment (abbas-nazar) [inline src/redux/ducks/comment.js:5]: why is this commented?

**Review by abbas-nazar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**